scripts/lola-perf.ps1

```diff
@@ -0,0 +1,85 @@
+<#
+
+.SYNOPSIS
+This script runs performance tests for LoLa and generates results in a table.
+
+.PARAMETER Binary
+    Specifies the build configuration to use.
+
+.PARAMETER Server
+    Specifies -target parameter for secnetperf.
```

Comment:
Why not call it Target then?

Reply:
Right

Reply:
Same comment for the ResponseSizes parameter; might as well give them the same name as the underlying options yeah?

Reply:
The param () part was updated but here it still says .PARAM Server